source/plugin.ts

```diff
@@ -104,7 +111,9 @@ const mochaPlugin: ESLint.Plugin = {
         'valid-suite-title': validSuiteTitleRule,
         'valid-test-title': validTestTitleRule,
         'no-empty-title': noEmptyTitleRule
-    }
+    },
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-explicit-any,@typescript-eslint/no-unsafe-type-assertion -- This field will be set later for cyclic references
+    configs: undefined as any
```

Comment:
This part is ugly but I could not find any other workaround for those cyclic references.